src/interface/swarm.cpp

```diff
@@ -35,6 +38,12 @@ SwarmDeviceContext Swarm::GetDeviceContext() const {
   const IndexRange &ib = pmb->cellbounds.GetBoundsI(IndexDomain::interior);
   const IndexRange &jb = pmb->cellbounds.GetBoundsJ(IndexDomain::interior);
   const IndexRange &kb = pmb->cellbounds.GetBoundsK(IndexDomain::interior);
+  context.ib_s_ = ib.s;
+  context.jb_s_ = jb.s;
+  context.kb_s_ = kb.s;
```

Comment:
I have ongoing concerns about the size of this context object, and since we don't need `*b.e`, I'd rather leave them out until we do need them (or the whole device context picture changes...)

Reply:
Yeah ok fair enough.